apis/apps/v1alpha1/type.go

```diff
@@ -1177,6 +1177,12 @@ type ComponentVars struct {
 	//
 	// +optional
 	PodFQDNs *VarOption `json:"podFQDNs,omitempty"`
+
+	// Specify whether add the k8s clusterdomain to pod FQDN.
+	// Add the clusterDomain to the pod FQDN
+	// if set true, fqdns will add k8s clusterDocmain.
+	// +optional
+	ClusterDomain *bool `json:"clusterDomain,omitempty"`
```

Comment:
It would be more appropriate to extend the definition of PodFQDNs rather than adding a separate API. Each item under the ComponentVars API is defined as an independent variable.